### PR TITLE
Improve CMS edit page footer - OLD SCREWED UP

### LINKED ISSUE #26633: [Mobile] BO - the save and preview buttons in Design>Pages are bad displayed
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

When you try to edit page, you can see that card-footer can not contain all buttons:
**PS1.7.8.0 and develop** :x: 

image

**PS1.7.7.8** :heavy_check_mark: 
image

### Expected behavior

Create dropdown-menu (like in Edit a Product page):

image

### Steps to reproduce

1. **BO**>**IMPROVE**>**Design**>**Pages**
2. Click on **Edit** button of any page
3. Scroll down to the **card-footer**
4. See error > the `Save` and `Save and preview` buttons are bad displayed

### PrestaShop version(s) where the bug happened

1.7.8.0 & develop

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | This PR turns two buttons into btn group on smaller screens. Drops up and left.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #26633
| How to test?      | Compare before and after.
| Possible impacts? | 

**Screenshot**
Multimédia1

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26704)

### CODE REVIEW

**Comment by prestascott** [comment]
Nice work @Hlavtox, LGTM!

**Comment by mparvazi** [comment]
@Hlavtox Great job!

**Review by NeOMakinG — DISMISSED** [review]

**Review by Julievrz — DISMISSED** [review]
Wording ✅ Thank you

**Review by Progi1984 — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @Hlavtox,

When I resize the window or I use a mobile, the button is well displayed but the "Save and preview" is not working.
I attached a screen record
https://watch.screencastify.com/v/WL8pEpAjUn7NM5YYcFxD

Thanks!

**Comment by Hlavtox** [comment]
@khouloudbelguith @NeOMakinG I need to see my eye doctor :-)))

`<btn type="submit" class="dropdown-item" name="save-and-preview" id="save-and-preview-button">{{ 'Save and preview'|trans({}, 'Admin.Actions') }}</btn>`

**Comment by NeOMakinG** [comment]
> https://watch.screencastify.com/v/WL8pEpAjUn7NM5YYcFxD

I think it's because some JS are linked to this "Save and preview" button and it's not added to both buttons (responsive + desktop one), don't you think?

**Comment by Hlavtox** [comment]
@NeOMakinG @khouloudbelguith It works fine now, it was just the element name wrong. ;-)

**Comment by NeOMakinG** [comment]
> @NeOMakinG @khouloudbelguith It works fine now, it was just the element name wrong. ;-)

Holy, didn't even notice it...

**Review by NeOMakinG — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/form.html.twig [hunk 1]
```diff
@@ -26,30 +26,39 @@
...
+      <button type="submit" class="btn btn-primary" name="save-and-preview" id="save-and-preview-button">
         {{ 'Save and preview'|trans({}, 'Admin.Actions') }}
       </button>
-
       <button type="submit" class="btn btn-primary ml-3" id="save-button">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
     </div>
+
+    <div class="d-lg-none ml-auto">
+      <div class="btn-group dropup">
+        <button type="submit" class="btn btn-primary" id="save-button">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
```
Comment (matks) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Cms/Blocks/form.html.twig]: I think it's a bad practice to have duplicate ID, isn't it?

**Review by matks — CHANGES_REQUESTED** [review]

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
There are duplicate ids :)

**Comment by Hlavtox** [comment]
@PierreRambaud @matks Yes, you are right. IDs are not needed for the functionality, but are needed for tests.

@boubkerbribri How do you want me to alternatively target the buttons without using IDs? Add `js-` class, add `data-role`?

**Comment by boubkerbribri** [comment]
> @boubkerbribri How do you want me to alternatively target the buttons without using IDs? Add `js-` class, add `data-role`?

For now, the Mobile views are not targeted by UI tests, I think you can delete the Mobile View buttons ids. We will find a solution when we will have to.

**Comment by Hlavtox** [comment]
@matks @PierreRambaud Duplicate ids fixed

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
`save-and-preview-button`  ids are still duplicate, watch out, you can see them in your browser developer tools :wink:

**Pull request closed without merging**